recipe/meta.yaml

```diff
@@ -10,46 +10,85 @@ source:
   patches:
 
 build:
-  number: 0
-  entry_points:
-    - aesara-cache = bin.aesara_cache:main
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 1
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - pip
-    - python
-    - blas
-    - libblas
-    - liblapack
   run:
-    - {{ compiler('c') }}  # [unix]
-    - {{ compiler('cxx') }}  # [unix]
-    - {{ compiler('m2w64_c') }}  # [win]
-    - {{ compiler('m2w64_cxx') }}  # [win]
-    - python
-    - setuptools
-    - six >=1.9.0
-    - numpy >=1.9.1
-    - scipy >=0.14
-    - filelock
-    - libpython >=2.0  # [win]
-    - dataclasses  # [py==36]
-    - logical-unification
-    - miniKanren
-    - cons
-    - etuples
-    - typing_extensions
 
 test:
   imports:
     - aesara
   commands:
     - aesara-cache help
 
+outputs:
+  - name: aesara-base
+    build:
+      script:
+        - python -m pip install . --no-deps -vv
+      entry_points:
+        - aesara-cache = bin.aesara_cache:main
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+      host:
+        - pip
+        - python
+        - blas
```

Comment:
I'm not sure even this (and the next two) are a hard dependency. Let's try moving them.

Reply:
Good point.

Host dependencies confuse me. My understanding is that they are essentially build dependencies, however in the case of cross-compiling, they reference the target system rather than the build system. They should not be runtime dependencies.

Looking in the commit history, `libblas` and `liblapack` were added by @brandonwillard in #27. I wonder what was the rationale behind that. If Aesara compiles some libraries at build time, and these libraries require BLAS to be full-featured, then we'll definitely want to add them back. However, if the goal was to add them as runtime dependencies, then I think this PR accomplishes that.

Reply:
I don't think libblas is needed for building aesara, but it's certainly needed for running (if the user wants to C-compile).

Reply:
But we do need them for `aesara` still.

Reply:
I checked each individual CI run, and in each case libblas and liblapack are being installed as implicit dependencies, presumably by mkl-service or openblas.

We could still add them as explicit runtime dependencies if that makes sense.

I wonder what the `blas` package does...

Reply:
Once we understand exactly what the requirements are, we may want to consult the conda-forge experts with blas experience to check this.

Here is the [advice from the knowledge base](https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=mesa#blas), but i don't think this is directly applicable to us since we require _runtime_ building against blas.

Reply:
This situation reminds me of [cmdstanpy](https://github.com/conda-forge/cmdstan-feedstock/blob/main/recipe/meta.yaml) but there we don't need blas.

Reply:
Yeah, `blas` is required for some odd reason, we've seen compile errors without it (https://discourse.pymc.io/t/pymc-4-0-0-b2-fatal-error-cblas-h/8791).

Reply:
Thanks, that's really helpful! In order to make this info easy to reference, I opened #80. I'm thinking we can explain our reasoning as comments in the feedstock and add such links as references. That way we don't rely so much on folklore.